modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,183 @@
+
+| `cleanup`
+| Remove netobserv components.
+
+| `version`
+| Print the software version.
+
+| `help`
+| Show help.
+|===
+
+== Feature commands
+The Network Observability Operator features can only be enabled when capturing flows. They do not work with packet capture.
+
+.Syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Network Observability feature enablement commands
+|===
+|Option| Description| Possible values| Default
+
+| `--help`
+| Show help
+| N/A
+| N/A
+
+| `--enable_pktdrop` 	
+| Enable packet drop
+| `true`, `false`
+| `false`
+
+| `--enable_rtt`
+| Enable round trip time 	
+| `true`, `false` 	
+| `false`
+
+| `--enable_dns`
+| Enable DNS tracking 	
+| `true`, `false` 	
+| `false`
+
+| `--interfaces`
+| Interfaces to match on the flow e.g., "eth0,eth1" 	
+| "<interface>"
+|===
+
+== Flow capture subcommands
+Flow capture has mandatory commands and optional commands. 
+
+.`oc netobserv flows` syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Flow capture subcommands
+|===
+|Option| Description| Possible values| Mandatory| Default
+
+| `--enable_filter`
+| Enable flow filter
+| `true`, `false` 	
+| yes 
+| `false`
+
+| `--action`
+| Action to apply on the flow
+| `Accept`, `Reject`
+| yes
+| `Accept`
+
+| `--cidr`
+| CIDR to match on the flow. 
+| `1.1.1.0/24`, `1::100/64`, or `0.0.0.0/0`
+| yes
+| `0.0.0.0/0`
+
+| `--protocol`
+| Protocol to match on the flow
+| `TCP`, `UDP`, `SCTP`, `ICMP`, `ICMPv6` 	
+| no
+| -
+
+| `--direction`
+| Direction to match on the flow
+| `Ingress`, `Egress`
+| no
+| -
+
+| `--dport`
+| Destination port to match on the flow
+| `80`, `443`, or `49051`
+| no
+| -
+
+| `--sport`
+| Source port to match on the flow
+| `80`, `443`, or `49051`
+| no
+| -
+
+| `--port`
+| Port to match on the flow
+| `80`, `443`, or `49051`
+| no 
+| -
+
+| `--sport_range`
+| Source port range to match on the flow
+| `80-100` or `443-445`
+| no 	
+| -
+
+| `--dport_range` 	
+| Destination port range to match on the flow
+| `80-100`
+| no
+| -
+
+| `--port_range`
+| Port range to match on the flow
+| `80-100` or `443-445`
+| no
+| -
+
+| `--icmp_type`
+| ICMP type to match on the flow
+| `8` or `13`
+| no
+| -
+
+| `--icmp_code`
+| ICMP code to match on the flow
+| `0` or `1`
+| no
+| - 	
+
+| `--peer_ip`
+| Peer IP to match on the flow
+| `1.1.1.1` or `1::1`
+| no 	
+| -
+|===
+
+== Packet capture subcommand
+You can filter on port and protocol for packet capture data. 
+
+.`oc netobserv packets` syntax
+[source,terminal]
+----
+oc netobserv packets [<subcommand>]
+----
+.Packet capture subcommand
+` --filter`:: A mandatory subcommand to enable packet capture filtering. Options for packet capture filtering are protocol (tcp, udp) and port. You can specify filtering options using a comma as delimeter. For example `--filter=tcp,80` specifies the `tcp` protocol and port `80`.
```

Comment:
The backticks are showing / not rendering as monospace.

However, I'd recommend putting this in a table, so that it's in the same format as the flow capture section. I realize there's only one, so it might seem overkill, but I think it'd be better to be consistent.

Also, should "tcp" and "udp" be in backticks?

Reply:
Okay I debated whether or not it was overkill too. I'll take your advice and do the table.